template/package.json

```diff
@@ -1,66 +1,48 @@
 {
-  "name": "widgets-template",
+  "name": "{{ name }}",
   "version": "0.1.0",
   "description": "",
+  "source": "src/index.js",
   "main": "dist/index.js",
+  "module": "dist/index.module.js",
   "umd:main": "dist/index.umd.js",
-  "module": "dist/index.m.js",
-  "source": "components/hello-world/index.js",
   "scripts": {
-    "test": "NODE_ENV=test jest",
-    "build": "preact build --no-prerender --clean --template src/index.ejs --service-worker false",
-    "serve": "preact serve",
-    "dev": "preact watch --template src/index.ejs",
-    "dist": "microbundle --entry src/components/hello-world/index.js"
+    "dev": "preact watch --template src/template.html",
+    "build:widget": "microbundle build",
+    "build:lib": "microbundle build -i src/component.js",
+    "lint": "eslint '{src,test}/**/*.js'",
+    "test": "jest"
+  },
+  "files": [
+    "dist"
+  ],
+  "eslintConfig": {
+    "extends": "preact",
+    "ignorePatterns": [
+      "build/"
+    ]
   },
-  "keywords": [],
   "author": "Zouhir <zouhir@zouhir.org>",
   "license": "MIT",
-  "devDependencies": {
-    "eslint": "^4.5.0",
-    "identity-obj-proxy": "^3.0.0",
-    "jest": "^21.2.1",
-    "microbundle": "^0.6.0",
-    "node-sass": "^4.5.3",
-    "preact-cli": "^2.0.2",
-    "preact-render-to-string": "^3.7.0",
-    "preact-widget-scripts": "^0.6.0",
-    "sass-loader": "^6.0.6"
-  },
   "dependencies": {
-    "preact": "^8.5.2",
-    "preact-compat": "^3.17.0",
-    "preact-habitat": "^3.1.0",
-    "preact-render-to-string": "^4.1.0",
-    "preact-router": "^2.5.7"
+    "preact": "^10.5.7"
```

Comment:
Should this be a peer dependency if used as a library?

Reply:
Ah, yeah, good point.